packages/devtools_app/lib/src/screens/deep_link_validation/deep_links_services.dart

```diff
@@ -90,8 +90,15 @@ class ValidateAndroidDomainResult {
 }
 
 class DeepLinksServices {
```

Comment:
naming nit: should this be `DeepLinksService` instead of the plural `DeepLinksServices`?